GoBoard.java

```diff
@@ -0,0 +1,139 @@
+import java.awt.*;
+import java.awt.event.*;
+import java.util.Random;
+import javax.swing.*;
+
+public class GoBoard{
+    public static void main(String[] args) {
+	// create a new frame to hold the panel
+        JFrame application = new JFrame();
+        // set the layout of the frame  
+        application.setLayout(new BorderLayout());
+	// add the panel to the frame
+        application.add(new GoPanel(19), BorderLayout.CENTER);
+		
+        application.setSize(600, 625);
+	// set the frame to exit when it is closed
+        application.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+	// make sure the size of the frame does not change
+        application.setResizable(false);
+	// make the frame visible
+        application.setVisible(true);
+    }
+}
+
+class GoPanel extends JPanel {
```

Comment:
Declaring two different classes within one file isn't really recommended in Java.